content/2020-06-05-pycones-2020-c4p-en.md

```diff
@@ -0,0 +1,101 @@
+Title: PyConES 2020 Call for Papers is open!!!
```

Comment:
```suggestion
Title: PyConES 2020 Call for Proposals is open!!!
```
I think "Proposals" makes more sense than "Papers" for the kind of submission that we ask for.